src/filtering/markov_kernel.jl

```diff
@@ -113,13 +113,35 @@ function marginalize_cov!(
 ) where {T,S}
     _Σ_out = PSDMatrix(Σ_out.R.B)
     _Σ_curr = PSDMatrix(Σ_curr.R.B)
-    _K = AffineNormalKernel(K.A.B, K.b, PSDMatrix(K.C.R.B))
+    _K = AffineNormalKernel(K.A.B, nothing, PSDMatrix(K.C.R.B))
```

Comment:
This is only good if we have an assert statement for K.b somewhere to check that it's actually empty.